tox.ini

```diff
@@ -48,11 +48,11 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION RUNNING_IN_CI
 commands =
     make update_deps
     pip install --exists-action=w -r tests/ui/requirements.txt
-    pytest --driver Firefox -v tests/ui {posargs}
+    pytest --boxed --driver Firefox -v tests/ui {posargs}
```

Comment:
So as I understand it, we're using boxed so that each test runs in a completely separate process, meaning the live server and fixtures are all independent of each other. We're seeing issues with running these processes in parallel when attempting to connect to geckodriver, so for now we're going to run them in serial. Let's raise issues for investigating the geckodriver issue when running in parallel (rather than try to debug it further here), and replacing `--boxed` with a session scoped `live_server` fixture that allows us to run tests in parallel.